### PR TITLE
duck 4.6.5

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: Library/Formula/duck.rb [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
+  url "https://dist.duck.sh/duck-src-4.6.5.16970.tar.gz"
+  sha1 "ea8c3e6af509f36ec122ed7669834ce9b725aeeb"
   head "https://svn.cyberduck.io/trunk/"
 
-  bottle do
-    cellar :any
-    sha1 "05353941783e34f1931b0cdd78a89f245d8daf33" => :yosemite
-    sha1 "a2120b7b4b7ae15f9b941c24bb9d4f7c36392e74" => :mavericks
-    sha1 "fb6e8e20d702cf3d9808e758328cd7ebc8830283" => :mountain_lion
-  end
```
Comment (mikemcquaid) [inline Library/Formula/duck.rb]: Just leave the bottle block as-is and we'll update it when we pull, thanks!

**Comment by mikemcquaid** [comment]
Still needs rebased, it seems. Rebase and squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master and then mark each commit other than the first as squash or fixup then git push -f. For future reference you can do this in the existing pull request and it'll be automatically updated (but don't worry about it for this time). Thanks!

**Comment by dkocher** [comment]
We have now modified our continuous build to pull changes from upstream before applying the changes. Also, the bottles will no longer be removed. As we have deleted and created a new branch this pull request got closed.

**Comment by dkocher** [comment]
Now in #37183.

**Pull request closed without merging**